airbyte-integrations/connectors/source-postgres-strict-encrypt/Dockerfile

```diff
@@ -16,5 +16,5 @@ ENV APPLICATION source-postgres-strict-encrypt
 
 COPY --from=build /airbyte /airbyte
 
-LABEL io.airbyte.version=0.4.15
+LABEL io.airbyte.version=0.4.16
```

Comment:
please bump the version, the latest actual in docker hub is [0.4.18](https://hub.docker.com/layers/source-postgres/airbyte/source-postgres/0.4.18/images/sha256-de3f1cd87721f399d808b998791a9477220d521b7986b50ba386d79013942600?context=explore)